src/dcmp/dcmp.c

```diff
@@ -1054,8 +1054,8 @@ static void dcmp_strmap_compare(mfu_flist src_list,
     mfu_flist dst_compare_list = mfu_flist_subset(dst_list);
   
     /* remove and copy lists for sync option */ 
-    mfu_flist dst_remove_list = MFU_FLIST_NULL;
-    mfu_flist src_cp_list     = MFU_FLIST_NULL; 
+    mfu_flist dst_remove_list = NULL;
+    mfu_flist src_cp_list     = NULL; 
```

Comment:
Let's keep with setting these to MFU_FLIST_NULL, but check for that instead of NULL in the if/free blocks below.